js/src/events/Main.js

```diff
@@ -123,7 +123,7 @@ const EventDetails = WithContext(({ctx, event, uri, ticket_types, existing_ticke
         </Col>
       ) : null}
 
-      {event.allow_donations ? (
+      {/* {event.allow_donations ? (
```

Comment:
Remove, rather than comment out.

If it might be needed in future, add an option and set it to false for now.